### PR TITLE
LPS-126213

### PR DESCRIPTION
**Pull request by alexcandidos**

### CODE REVIEW

**Comment by alexcandidos** [comment]
Hi @Tim-Cao , thanks for the review! Discussing with @emanuelfisilva , we'll keep them as is in the Data Engine scope.

**Comment by Tim-Cao** [comment]
Got it

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase [hunk 1]
```diff
@@ -25,13 +25,35 @@ definition {
 	}
 
 	@description = "This is a test for LPS-123539. As Content Creator, I want to leverage the structures’ translations with fildset when creating a new Web Content"
-	@ignore = "true"
 	@priority = "5"
 	test CheckDefaultWebContentTitleOnWebContentList {
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase:29]: As Content Creator, I want to leverage the structures’ translations with fieldset when creating a new Web Content
The steps are inconsistent with description

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase [hunk 2]
```diff
@@ -25,13 +25,35 @@ definition {
...
 
-		// TODO LPS-123539 CheckDefaultWebContentTitleOnWebContentList pending implementation
+		Button.clickSave();
 
+		NavItem.click(navItem = "Web Content");
+
+		WebContentNavigator.gotoAddWithStructureCP(structureName = "${structureTitle}");
+
+		WebContent.addWithStructureCP(webContentTitle = "${webTitle}");
+
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase:53]: Should add steps to add localized content

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3633
View total diff https://github.com/liferay-echo/liferay-portal/pull/3633/commits/43feba67c23ef16b68bf8798014c7c547443682c

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**